src/Field/Field.tsx

```diff
@@ -56,13 +56,15 @@ export const Field = ({
           )}
         </Box>
       )}
-      <Box>
-        {assistiveText && (
-          <Text tag="p" typo="body" color="subtext">
+
+      {assistiveText && (
+        <Box>
+          <Text tag="p" color="subtext">
             {assistiveText}
           </Text>
-        )}
-      </Box>
+        </Box>
```

Comment:
do we actually need the Box at all? Have you also checked that margins are fine? I also believe that this case will only apply if we have `renderAsTitle`? wondering if we should combine them cause I don't think we'll ever have a case were we have `assistiveText` and no `renderAsTitle`

Reply:
Moved it into the `renderAsTitle` conditional in [616b2a](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/616b2a5e21517f733b6649623a18b1642444226d)